### ISSUE TITLE
Errors that cause HHVM to crash

### ISSUE THREAD

**Issue by tomschlick** [ptrofimov/beanstalk_console#49]
These show up in the logs every time i refresh the page using HHVM. When this happens only the toolbar will show up in the output, the stats is blank. After a restart of HHVM it will work for a while and then the same cycle happens again.

```bash
Notice: Undefined variable: contentType in /path/to/dir/lib/tpl/main.php on line 15
Notice: Undefined variable: _tplPage in /path/to/dir/lib/tpl/main.php on line 132
Warning: File not found:/path/to/dir/lib/tpl/.php in /path/to/dir/lib/tpl/main.php on line 132
```

**Comment by pentium10** [comment]
Are you able to check the newest build and see if the errors are still reported?

**Comment by pentium10** [comment]
Lots of compatibility issues were fixed with bd2aad0ab5efd8cb345a04042948551da3141204
